lib/mtg_sdk/representers/card_representer.rb

```diff
@@ -41,16 +41,17 @@ module CardRepresenter
     property :set_name, as: :setName
     property :id
     property :image_url, as: :imageUrl
-    
+
     collection :names
     collection :supertypes
     collection :subtypes
     collection :types
     collection :colors
+    collection :color_identities, as: :colorIdentity
```

Comment:
Can you call this `color_identity` to match the other sdks and the API? Thank you!